### DISCUSSION TITLE
Reorganize Sections on “Home” Screen”

### DISCUSSION THREAD

**Discussion by Gta211** [community/community#144302] [Mobile]
Feature Request : Please add the option to reorganize the "My Work", "Favorites","Shortcut's" & "Recent" sections on the "Home" Tab on the mobile app on iOS. This would be really nice to see. Please!!
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.185.0 (146370144)
Device: iPhone 14 Pro Max (iOS 18.0.1)
</details>

**Comment by kangarruu** [comment]
@Gta211 Thanks for writing in, I filed this request with our team for future design exploration! Hopefully we could add something like this to our roadmap soon ❤️
